app/consumers/compliance_reports_consumer.rb

```diff
@@ -14,6 +14,9 @@ def process(message)
     logger.error "Error parsing report: #{@msg_value['request_id']}"\
       " - #{e.message}"
     send_validation('failure')
+  ensure
+    @file.close
```

Comment:
You can delete a file that's not closed without error. This line isn't actually required.